surface/model_openapiv3.go

```diff
@@ -185,40 +233,75 @@ func (b *OpenAPI3Builder) buildTypeFromParameters(
 			}
 			f.Serialize = true
 			t.addField(&f)
+		} else if parameterRef := parametersItem.GetReference(); parameterRef != nil {
+			f.Type = typeForRef(parameterRef.GetXRef())
+			f.Name = strings.ToLower(f.Type)
+			f.Serialize = false
```

Comment:
This is unnecessary because the default value for a `bool` is `false`.

Reply:
Done